### ISSUE TITLE
Setting Nodes data depending on variables/parameters.

### ISSUE THREAD

**Issue by C1ffisme** [minetest/minetest#3320]
Currently, to do something with many nodes, you need to define every single different type. You could have a mod like mesecons where you need a bunch of nodes for each connection, or like wool where you have several colors and you need a node for each one of them.

In minecraft, however, wool is one node. All wool is just one node, but colored by a data value. Why can't minetest do this? This would save on server load time, as well as node count. We can do this with entities right now. Inside entity code, you can set the texture and model of the mob.

Without this:
![without](https://cloud.githubusercontent.com/assets/9893266/10864499/d17361da-7fbd-11e5-9b4f-b531dc8b6990.png)
Total number of nodes : 4

With this:
![with](https://cloud.githubusercontent.com/assets/9893266/10864502/dd5d38e0-7fbd-11e5-8cb7-b145831b245b.png)
Total number of nodes : 1

**Comment by rubenwardy** [comment]
Duplicate of #2887 which is solved by #1118

**Comment by C1ffisme** [comment]
I think we need a database for common ideas and their feature request issues, so that we don't make a thousand duplicates. People can look at the database and figure out where the request issue is.

**Comment by PilzAdam** [comment]
@C1ffisme what you are talking about is done in this issue tracker.

**Comment by C1ffisme** [comment]
I'm talking about a page with a table with descriptions written in plain english, and then in the second collumn a link to that issue.

**Comment by rubenwardy** [comment]
like https://github.com/minetest/minetest/issues

**Comment by C1ffisme** [comment]
Not all of the descriptions are in plain English. A searcher should be able to search many different versions of a sentence in case the sentence in their mind isn't the same as the actual issue.

**Comment by rubenwardy** [comment]
you don't search sentences, you search keywords. For example "appearance", it's the second option: https://github.com/minetest/minetest/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+appearance

**Comment by rubenwardy** [comment]
you don't search sentences, you search keywords. For example "appearance", it's the second option: https://github.com/minetest/minetest/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+appearance
or even https://github.com/minetest/minetest/issues?utf8=%E2%9C%93&q=is%3Aopen+change+node+data

We accept that duplicates will be made. Don't feel embarrassed. It's why we close and label them.

**Comment by C1ffisme** [comment]
I'm not embarrassed, just leaving a suggestion. :wink:
